Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -205,6 +206,38 @@ class KillOldProcesses(shell.Compile):
     command = ["python3", "Tools/CISupport/kill-old-processes", "buildbot"]
 
 
+class PruneCoreSymbolicationdCacheIfTooLarge(buildstep.BuildStep):
+    name = "prune-coresymbolicationd-cache-if-too-large"
+    description = ["pruning coresymbolicationd cache to < 10GB"]
+    descriptionDone = ["pruned coresymbolicationd cache"]
+    flunkOnFailure = True
+    haltOnFailure = True
+
+    def __init__(self):
+        buildstep.BuildStep.__init__(self)
+
+    def start(self):
+        directory = '/System/Library/Caches/com.apple.coresymbolicationd'
+        directoryPath = Path(directory).resolve()
```

Comment:
Does this actually operate on the worker? cc @aj062 to double check that If so, I _love_ the approach!

Reply:
Indeed, let's confirm that! I suspect that it's probably running on the master.

Reply:
Yes, I think it would run on master.
For example this email code (`send_email_for_git_issue`) in `start()` also run on server: https://github.com/WebKit/WebKit/blob/main/Tools/CISupport/ews-build/steps.py#L658